_ml-commons-plugin/index.md

```diff
@@ -23,3 +23,12 @@ There are two reserved user roles that can use of the ML commons plugin.
 
 - `ml_full_access`: Full access to all ML features, including starting new ML tasks and reading or deleting models.
 - `ml_readonly_access`: Can only read ML tasks, trained models and statistics relevant to the model's cluster. Cannot start nor delete ML tasks or models.
+
+## ML node
+
+To prevent your cluster from failing when running ML tasks, you configure a node with the `ml` node role. When configuring without the `data` node role, ML nodes will not store any shards and calculates resource requirements at runtime. To use an ML node, create a node in your `opensearch.yml` file. Change the name of the node to `ml-node` and define the node role as `ml`:
```

Comment:
Second sentence: Should it be "...ML nodes will not store any shards and will calculate..."?